test/variants/seman/packing_app01.v3

```diff
@@ -0,0 +1,3 @@
+//@seman = TypeError @ 3:24
```

Comment:
I would split this test into three; one that passes, one that fails on the first argument, and one that fails on the second argument, which makes sure that the compiler is performing type checking and it is checking all fields.